test/checkstruct_align1.h

```diff
@@ -0,0 +1,1360 @@
+    SDL_FingerID fingerID;
+    float x;            
+    float y;            
+    float dx;           
+    float dy;           
+    float pressure;     
+    SDL_WindowID windowID; 
+} SDL_TouchFingerEvent_pack1;
+
+typedef struct SDL_PenProximityEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_WindowID windowID; 
+    SDL_PenID which;        
+} SDL_PenProximityEvent_pack1;
+
+typedef struct SDL_PenMotionEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_WindowID windowID; 
+    SDL_PenID which;        
+    SDL_PenInputFlags pen_state;   
+    float x;                
+    float y;                
+} SDL_PenMotionEvent_pack1;
+
+typedef struct SDL_PenTouchEvent_pack1 {
+    SDL_EventType type;     
+    Uint32 reserved;
+    Uint64 timestamp;       
+    SDL_WindowID windowID;  
+    SDL_PenID which;        
+    SDL_PenInputFlags pen_state;   
+    float x;                
+    float y;                
+    Uint8 eraser;           
+    Uint8 state;            
+} SDL_PenTouchEvent_pack1;
+
+typedef struct SDL_PenButtonEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_WindowID windowID; 
+    SDL_PenID which;        
+    SDL_PenInputFlags pen_state;   
+    float x;                
+    float y;                
+    Uint8 button;       
+    Uint8 state;        
+} SDL_PenButtonEvent_pack1;
+
+typedef struct SDL_PenAxisEvent_pack1 {
+    SDL_EventType type;     
+    Uint32 reserved;
+    Uint64 timestamp;       
+    SDL_WindowID windowID;  
+    SDL_PenID which;        
+    SDL_PenInputFlags pen_state;   
+    float x;                
+    float y;                
+    SDL_PenAxis axis;       
+    float value;            
+} SDL_PenAxisEvent_pack1;
+
+typedef struct SDL_DropEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_WindowID windowID;    
+    float x;            
+    float y;            
+    const char *source; 
+    const char *data;   
+} SDL_DropEvent_pack1;
+
+typedef struct SDL_ClipboardEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+} SDL_ClipboardEvent_pack1;
+
+typedef struct SDL_SensorEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_SensorID which; 
+    float data[6];      
+    Uint64 sensor_timestamp; 
+} SDL_SensorEvent_pack1;
+
+typedef struct SDL_QuitEvent_pack1 {
+    SDL_EventType type; 
+    Uint32 reserved;
+    Uint64 timestamp;   
+} SDL_QuitEvent_pack1;
+
+typedef struct SDL_UserEvent_pack1 {
+    Uint32 type;        
+    Uint32 reserved;
+    Uint64 timestamp;   
+    SDL_WindowID windowID; 
+    Sint32 code;        
+    void *data1;        
+    void *data2;        
+} SDL_UserEvent_pack1;
+
+typedef union SDL_Event_pack1 {
+    Uint32 type;                            
+    SDL_CommonEvent common;                 
```

Comment:
Struct members need replacements too.
Nothing a regex can't solve ;)

Reply:
Go for it. ;)

Reply:
Tag me when you're finished, so we don't step on each other's toes :)